Cabal/doc/nix-local-build.rst

```diff
@@ -357,6 +357,16 @@ A target can take any of the following forms:
      - ``tests``,
      - ``benches``, ``benchmarks``.
 
+-  A module target: ``[package:][ctype:]module``, which specifies that the
+   component of which the given module is a part of will be built. If the
+   target is ambiguous, e.g. only ``module`` is specified and the module name
+   is part of multiple components, then no component will be built.
+
+-  A filepath target: ``[package:][ctype:]filepath``, which specifies that the
+   component of which the given filepath is a part of will be built. If the
+   target is ambiguous, e.g. only ``filepath`` is specified and the filepath
+   is part of multiple components, then no component will be built.
```

Comment:
This needs to be proven with a test.
Currently, an ambiguous target produces this error message:
```
TargetSelector: internal error: NoMatch
CallStack (from HasCallStack):
  error, called at .\\Distribution\\Client\\TargetSelector.hs:669:3 in main:Distribution.Client.TargetSelector
```